script_vts.py

```diff
@@ -202,95 +207,84 @@ def formating_file():
             out_file.truncate()
     except IOError as strip_err:
         exception_handler(True, strip_err)
-        exit(1)
 
     return
 
 
-if __name__ == '__main__':
+def logic_func():
```

Comment:
We need to improve this name so even non-technical readers can understand what the function is likely doing.

If you find this function hard to name more descriptively, it's a good indicator that the function is unfocused an needs a re-factor. We can look into this in 1-2 iterations if you don't have a good idea how to proceed here.

Reply:
I thought about handle_report. More clear but indeed we'll need to make some change and move code after some iterations. But basically it's way clearer to put a name like this than logic func. Because basically we're handling a report from the logic, appending, creating etc etc. So for now this function is a handle I'd say ?